### DISCUSSION TITLE
Airflow 2.4.2 Grid View disappears when clicking dag step button

### DISCUSSION THREAD

**Discussion by cho6646** [apache/airflow#28185] [Q&A] [answered]
Hi, our team has recently updated our airflow to version 2.4.2.

Then we got an issue that grid view page disappear when clicking dag step button (see below)
<img width="1214" alt="iShot_2022-12-07_17 13 35" src="https://user-images.githubusercontent.com/20199350/206125234-264c2505-e232-4870-b853-115d72142e73.png">

The grid view disappears with message (typeerror: n.split is not a function)
<img width="2567" alt="iShot_2022-12-07_17 18 54" src="https://user-images.githubusercontent.com/20199350/206125632-1ec77cdf-4a64-4dd3-98e1-8360ddf87982.png">

I think there is an issue regarding the view with message that should show up when clicking the button. Some variable (not set as string) was used with string.split function.

Anyone has an idea regarding this or has similar issues?

Thank you!

**Comment by potiuk** [comment, marked as answer]
I believe there were similar issues solved in subsequent releases (we are on 2.5.0 now with more than 100 fixes/changes comparing to your version). 

I cannot pin-point it to exact issue, but I recall similar discussions. You can look in our https://github.com/apache/airflow/issues or https://github.com/apache/airflow/pr for similar problems if you want to be sure. But since the only way to fix it will be to upgrade anyway - you should upgrade and see if you see the same problem (and report it as an issue with some more details (like what kind of  dag causes it).  Upgrading is the fastest way to check if it has been fixed (both for you and for people who would like to help you).

**Reply by cho6646** [reply]
thank you!! we will check!
